packs/pfs-season-1-bestiary/deeply-flawed-ritual.json

```diff
@@ -67,7 +67,7 @@
                 "title": "Pathfinder Society Scenario #1-01: The Absalom Initiation"
             },
             "reset": "",
-            "routine": "<p>(1 action) On each of its turns, the ritual lets out a surge of shadowy energy, dealing @Damage[4d6[void]] damage (@Check[fortitude|dc:22|basic|name:Flawed Ritual] save) to each character who participated in ritual the past round (including Tavvar and her aides as well as any PC who attempted a skill check to disable the hazard). A non-participant can step in front of a participant as a reaction. This blocks the damage from the ritual caster, but the non-participant must then attempt the Fortitude save.</p>\n<p>In addition to dealing damage, the ritual's surge of energy brings a @UUID[Compendium.pf2e.pfs-season-1-bestiary.Actor.Greater Shadow Wisp] into the room. Wisps act at the end of the ritual's turn. If there are already two shadow wisps in the room, instead of creating a new wisp, the ritual restores [[/r 10[healing]]]{10 Hit Points} to each wisp.</p>"
+            "routine": "<p>(1 action) On each of its turns, the ritual lets out a surge of shadowy energy, dealing @Damage[4d6[void]] damage (@Check[fortitude|dc:22|basic|name:Flawed Ritual] save) to each character who participated in ritual the past round (including Tavvar and her aides as well as any PC who attempted a skill check to disable the hazard). A non-participant can step in front of a participant as a reaction. This blocks the damage from the ritual caster, but the non-participant must then attempt the Fortitude save.</p><p>In addition to dealing damage, the ritual's surge of energy brings a @UUID[Compendium.pf2e.pfs-season-1-bestiary.Actor.Greater Shadow Wisp] into the room. Wisps act at the end of the ritual's turn. If there are already two shadow wisps in the room, instead of creating a new wisp, the ritual restores @Damage[10[healing]]{10 Hit Points} to each wisp.</p>"
```

Comment:
```suggestion
            "routine": "<p>(1 action) On each of its turns, the ritual lets out a surge of shadowy energy, dealing @Damage[4d6[void]] damage (@Check[fortitude|dc:22|basic|name:Flawed Ritual] save) to each character who participated in ritual the past round (including Tavvar and her aides as well as any PC who attempted a skill check to disable the hazard). A non-participant can step in front of a participant as a reaction. This blocks the damage from the ritual caster, but the non-participant must then attempt the Fortitude save.</p><p>In addition to dealing damage, the ritual's surge of energy brings a @UUID[Compendium.pf2e.pfs-season-1-bestiary.Actor.Greater Shadow Wisp] into the room. Wisps act at the end of the ritual's turn. If there are already two shadow wisps in the room, instead of creating a new wisp, the ritual restores @Damage[10[healing]] Hit Points to each wisp.</p>"
```